build.sc

```diff
@@ -28,6 +28,20 @@ trait CommonPublish extends CiReleaseModule with Mima {
 
   override def mimaPreviousVersions = Seq("1.5.4")
 
+  override def mimaBinaryIssueFilters =
```

Comment:
Got back to this since it's more important for Scala CLI as Scala and overall makes sense.

I propose that we just do 2.0.0 and just not worry to much about keeping 1.x forever. Binary compatibility is important, but I don't see a lot of tools using each other so maybe it's fine to just release new major